packages/core/src/modules/credentials/formats/jsonld/JsonLdCredentialFormat.ts

```diff
@@ -0,0 +1,27 @@
+import type { W3cCredential } from '../../../vc/models/credential/W3cCredential'
+import type { CredentialFormat } from '../CredentialFormat'
+import type { JsonLdOptionsRFC0593 } from './JsonLdOptionsRFC0593'
+
+export interface JsonLdAcceptRequestOptions extends SignCredentialOptionsRFC0593 {
+  verificationMethod?: string
+}
+
+// feel free to rename this, the RFC suffix is useful during dev
+
+export interface SignCredentialOptionsRFC0593 {
+  credential: W3cCredential
+  options: JsonLdOptionsRFC0593
+}
+
+export interface JsonLdCredentialFormat extends CredentialFormat {
+  formatKey: 'jsonld'
+  credentialRecordType: 'w3c'
+  credentialFormats: {
+    createProposal: SignCredentialOptionsRFC0593
+    acceptProposal: SignCredentialOptionsRFC0593
+    createOffer: SignCredentialOptionsRFC0593
+    acceptOffer: SignCredentialOptionsRFC0593
+    createRequest: SignCredentialOptionsRFC0593
+    acceptRequest: JsonLdAcceptRequestOptions
```

Comment:
Should these all be the same for each call? E.g. if we have the acceptProposal call, we should probably only need to provide some values. Otherwise there's no different between the `acceptProposal` and `createOffer` methods.

Reply:
I'm not sure how to resolve this one. The two methods acceptProposal and createOffer are identical but I can't see where the values provided are any different either, otherwise I could pass in a different interface.

Thoughts?

Reply:
I can't see any way to change this.